k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -290,28 +289,6 @@ function deleteKafkaZookeper()
     helm uninstall zookeeper -n "default"
 }
 
-function deleteOpenLdap()
```

Comment:
I think you would want to keep this and the other stuff for a couple releases. Someone who uses this script on an existing cluster with openldap won't have it cleaned up.

There are other examples which do the same, and ignore errors if not missing. I would look to them.

Reply:
Just ran into this actually, making a patch on top of this PR and deleting the cluster stalled when deleting PVCs. The script hung until I manually uninstalled the chart (and the pods deleted).

Reply:
I agree.  (And I saw the same thing you did, once I started testing.)  I have updated this PR so that it will silently delete openldap resources if they are present (statefulset, pvcs, pvs, services).  It's not completely silent -- the helm uninstall leaves a little message.  But for the normal case where openldap is not installed, it silently skips over it.

Thanks for the comment... this is a good change.

(This also gives me ideas for additional regression tests.  It should be straightforward to script up deploying / destroying different versions.)